### ISSUE TITLE
Similar list on github

### ISSUE THREAD

**Issue by jackstine** [Romaixn/awesome-communities#16]
Here is a similar list on github, but this community was easier to find via google search.

Maybe mix the two...
https://github.com/thisdot/tech-community-slacks

**Comment by Romaixn** [comment]
Yeah maybe we can link this GitHub in this one? tech-community-slacks looks very specific to Slack and not discord or others software
